ctx.go

```diff
@@ -1978,3 +1978,9 @@ func (c *DefaultCtx) setMatched(matched bool) {
 func (c *DefaultCtx) setRoute(route *Route) {
 	c.route = route
 }
+
+func (c *DefaultCtx) Drop() error {
+	conn := c.RequestCtx().Conn()
```

Comment:
this can be one line

return c.RequestCtx().Conn().Close()

Reply:
please check again